src/app/clients/add/formSections/AddressCard.tsx

```diff
@@ -2,6 +2,7 @@ import React from "react";
 import FreeFormTextInput from "@/components/DataInput/FreeFormTextInput";
 import { CardProps, errorExists, errorText, onChangeText } from "@/components/Form/formFunctions";
 import GenericFormCard from "@/components/Form/GenericFormCard";
+import { FormGap } from "@/components/Form/formStyling";
 
 const postcodeRegex =
     /^([Gg][Ii][Rr] 0[Aa]{2})|((([A-Za-z][0-9]{1,2})|(([A-Za-z][A-Ha-hJ-Yj-y][0-9]{1,2})|(([A-Za-z][0-9][A-Za-z])|([A-Za-z][A-Ha-hJ-Yj-y][0-9][A-Za-z]?))))\s?[0-9][A-Za-z]{2})$/;
```

Comment:
This isn't actually my ticket! I changed a couple bits on it but only when I had to because I changed shared functions. My ticket is just the add parcels page. In theory I agree though.